app/assets/stylesheets/application.css

```diff
@@ -67,6 +67,10 @@ a {
   text-align: left;
 }
 
+.my-1 {
+  margin: 10px 0;
```

Comment:
### GOOD JOB MAN, I LIKE YOUR PROJECT 👏🏼💥✊🏼👍🏼👋🏼👌🏼🎉🙌🏼. 

- [ ] [OPTIONAL] Please change license to MIT and also add the MIT.md license file 🍀